src/components/DropdownMenu/DropdownMenu.js

```diff
@@ -11,6 +11,29 @@ export default class DropdownMenu extends React.Component {
 
   someDescription;
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: this.props.active,
+    };
+  }
+
+  _onClickOutside = (e) => {
+    if (e.target !== document.body && e.target !== null) {
```

Comment:
Wouldn't this close the DrawerMenu even if the click happens on a blank space inside the DropdownMenu?